File: Data Structures For Interview/Java/Array Programs/DuplicateElements.java
class DuplicateElemnts{

    static void duplicateElements(int arr[]){

        int array[]=new int[arr.length];
        
        for(int i=0;i<array.length;i++){
            array[i]=0;
        }

        for(int i=0;i<arr.length;i++){

            int element=arr[i];
            int count=0;
            int flag=0;

            for(int k=0;k<array.length;k++){
                if(array[k]==element){
                    flag=1;
                    break;
                }
            }

            if(flag==0){
                for(int j=i+1;j<arr.length;j++){
                    if(arr[j]==element){
                        count++;
                        break;
                    }
                }
    
                if(count>0){
                    System.out.println(element+" is duplicate element");
                }
            }

            
            count=0;
            array[i]=element;
        }
    }

    public static void main(String s[]){

        int arr[]={10,20,30,40,50,60,70,80,90,100,10,50,60};

        duplicateElements(arr);
    }
}

File: Data Structures For Interview/Python/Array Programs/DuplicateElements.py
def duplicateElements(list1):

    list2=list(range(len(list1)))
    index=0

    for i in range(0,len(list2)):
        list2[i]=0

    for i in range(0,len(list1)):
        
        element=list1[i]
        count=1
        flag=0

        for k in range(0,len(list2)):
            if list2[k]==element:
                flag=1
                break
        
        if flag==0:

            for j in range(i+1,len(list1)):
                if list1[j]==element:
                    count=count+1

            if count>1:
                print(element," is duplicate element ")
            
            list2[index]=element
            index=index+1

list1=[1,2,3,4,5,5,6,7,8,9,10,10,11,11,12,13,12,13,14,15,15,14]
duplicateElements(list1)